api/v2010/application/create-default/output/create-default.json

```diff
@@ -3,7 +3,7 @@
   "api_version": "2010-04-01",
   "date_created": "Mon, 22 Aug 2011 20:59:45 +0000",
   "date_updated": "Tue, 18 Aug 2015 16:48:57 +0000",
-  "friendly_name": "Application Friendly Name",
+  "friendly_name": "friendly_name",
   "message_status_callback": "http://www.example.com/sms-status-callback",
```

Comment:
Not sure what's happening here, but I don't think it's right.

Reply:
nice catch. Upon investigating, it looks like we were using a default string for those values, but now we are just using the parameter name instead, in order to follow with the conventions of the rest of the languages since they are all using the same base convention to generate default values.